Apply fix from 38d582ff541353d738858299d4a2b78bafac03ed to unbreak
building with --disable-manual.
We don't use the literal patch since that means re-generating files
from the tarball that are usually shipped.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura curl-7.36.0.orig/src/tool_hugehelp.c curl-7.36.0/src/tool_hugehelp.c
--- curl-7.36.0.orig/src/tool_hugehelp.c	2014-03-31 19:28:23.018515093 -0300
+++ curl-7.36.0/src/tool_hugehelp.c	2014-03-31 19:28:09.994076200 -0300
@@ -4165,6 +4165,10 @@
 "  these mailing lists instead of mailing any individual.\n"
 , stdout) ;
 }
+#else /* !USE_MANUAL */
+/* built-in manual is disabled, blank function */
+#include "tool_hugehelp.h"
+void hugehelp(void) {}
 #endif /* USE_MANUAL */
 #else
 /*
@@ -8192,5 +8196,9 @@
   }
   inflateEnd(&z);
 }
+#else
+/* built-in manual is disabled, blank function */
+#include "tool_hugehelp.h"
+void hugehelp(void) {}
 #endif /* USE_MANUAL */
 #endif /* HAVE_LIBZ */
